mypyc/transform/exceptions.py

```diff
@@ -77,14 +79,21 @@ def split_blocks_at_errors(blocks: List[BasicBlock],
                 elif op.error_kind == ERR_NEG_INT:
                     variant = Branch.NEG_INT_EXPR
                     negated = False
+                elif op.error_kind == ERR_ALWAYS:
+                    variant = Branch.BOOL_EXPR
+                    negated = True
+                    tmp = LoadInt(0, rtype=bool_rprimitive)
+                    cur_block.ops.append(tmp)
+                    target = tmp
```

Comment:
here's my attempt to add a tmp between the CallC and the generated branch. I've made the branch to reference this tmp's value when doing the comparison. However, it still fails at refcount transform pass:
```
  File "/Users/th3charlie/Documents/mypy/mypyc/transform/refcount.py", line 181, in insert_branch_inc_and_decrefs
    source_borrowed, source_live_regs, env, omitted),
  File "/Users/th3charlie/Documents/mypy/mypyc/transform/refcount.py", line 210, in after_branch_decrefs
    for reg in sorted(decref, key=lambda r: env.indexes[r])
  File "/Users/th3charlie/Documents/mypy/mypyc/transform/refcount.py", line 210, in <lambda>
    for reg in sorted(decref, key=lambda r: env.indexes[r])
KeyError: <mypyc.ir.ops.LoadInt object at 0x10700cdd0>
```
The keyerror is due to not adding the tmp op into the environment.